types/abeeway/badgeTracker/meta.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Badge Tracker",
+  "version": "1.0.0",
+  "manufacturer": "Abeeway",
+  "url": "https://www.abeeway.com/smart-badge/",
+  "description": "A multi-mode tracker with embedded sensors combining multi-constellation GNSS, Low-power GPS, Wi-Fi Sniffer, BLE and LoRaWAN®TDoA geolocation technologies, for seamless outdoor and indoor geolocation.",
+  "author": "Akenza AG",
+  "firmwareVersion": "V2.1.6",
+  "loraDeviceClass": "A",
+  "availableSensors": ["GPS"],
+  "outputTopics": [
+    "acknowledge",
```

Comment:
I think this is definitely too many topics. Can you try to separate the messages into control and data messages and for the control ones you use the lifecycle topic?